src/get-transaction-path.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = (transaction) ->
+  origin = transaction['origin']
+  delimiter = ":"
```

Comment:
Since the API Blueprint specification doesn't disallow users to use `:` when naming things, I'd go for something else than ':'

https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#identifier

Maybe `#` - because it is otherwise used for markdown headers, thus not used inside the name of Resource, Group, Action?

cc. @pksunkara @zdne